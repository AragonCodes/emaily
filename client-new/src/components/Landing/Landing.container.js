import { connect } from 'react-redux';
import { withRouter } from 'react-router-dom';
import LandingComponent from './Landing';

const mapStateToProps = ({ auth }) => {
	return { auth };
}
const LandingContainer = connect(
	mapStateToProps
)(withRouter(LandingComponent));

export default LandingContainer;