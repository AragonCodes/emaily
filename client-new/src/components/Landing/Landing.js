import React from 'react';

function Landing({ auth, history }) {
	if (auth) {
		history.push('/surveys');
	}
	return (
		<div style={{ textAlign: 'center' }}>
			<h1>
				Emaily!
			</h1>
			Collect feedback form your users
		</div>
	);
}

export default Landing;