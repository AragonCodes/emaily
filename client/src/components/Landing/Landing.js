import React from 'react';
import MailLogo from '../../assets/mail_white.svg';

function Landing({ auth, history }) {
	if (auth) {
		history.push('/surveys');
	}
	return (
		<div className="center">
			<div className="red accent-2" style={{ borderRadius: '50%', padding: '3rem', width: 'fit-content', margin: ' 3rem auto 0' }}>
				<img src={MailLogo} style={{ height: '10rem' }} />
			</div>
			<h1>Emaily!</h1>
			<h4>Collect feedback form your users</h4>
		</div>
	);
}

export default Landing;