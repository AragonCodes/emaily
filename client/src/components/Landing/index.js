import LandingContainer from './Landing.container';

export default LandingContainer;