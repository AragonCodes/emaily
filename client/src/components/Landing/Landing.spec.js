import React from 'react';
import { shallow } from 'enzyme';
import Landing from './Landing';

describe('Landing', () => {
	it('should render correctly', () => {
		const wrapper = shallow(
			<Landing
				auth={null}
				history={[]}
			/>
		);
		expect(wrapper.debug()).toMatchSnapshot();
	});
});