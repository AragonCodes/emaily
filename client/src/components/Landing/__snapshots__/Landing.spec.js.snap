// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing should render correctly 1`] = `
"<div className=\\"center\\">
  <div className=\\"red accent-2\\" style={{...}}>
    <img src=\\"mail_white.svg\\" style={{...}} />
  </div>
  <h1>
    Emaily!
  </h1>
  <h4>
    Collect feedback form your users
  </h4>
</div>"
`;
